.github/workflows/test.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
       - name: Install Cargo Expand
         uses: taiki-e/install-action@v2
         with:
-          tool: cargo-expand
+          tool: cargo-expand@1.0.80
```

Comment:
This version, along with the versions in the cargo-make makefiles(including the other pinned binary dependencies) should be auto-updated by pipelines (similar to other things managed by dependabot). It looks like this is [not supported by dependabot yet](https://github.com/dependabot/dependabot-core/issues/9420), but there's another popular open source tool called [renovatebot that supports this](https://docs.renovatebot.com/modules/manager/regex/).